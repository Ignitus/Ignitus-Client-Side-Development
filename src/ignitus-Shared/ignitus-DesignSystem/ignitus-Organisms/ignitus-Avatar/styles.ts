import styled from '@emotion/styled';

export const AvatarImage = styled.img`
  max-width: 9rem;
`;
