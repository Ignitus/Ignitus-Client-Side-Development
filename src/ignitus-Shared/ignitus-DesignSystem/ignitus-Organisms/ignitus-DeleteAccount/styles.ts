import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import { Paragraph } from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';
import { Icon } from '../../../ignitus-Utilities/Components/icon';
import { DefaultCheckbox } from '../../..';
import { RoundedButton } from '../../ignitus-Atoms/buttons';

export const DeleteContainer = styled.div`
  background-color: ${C.White};
  background-size: contain;
  border-radius: 1rem;
  padding: 2rem;
  display: inline-flex;
  flex-direction: column;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  width: 40rem;
`;

export const TopSection = styled(flexibleRowDiv)`
  justify-content: space-between;
`;

export const MiddleSection = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
`;

export const BottomSection = styled(flexibleRowDiv)`
  justify-content: flex-end;
  margin-top: 2rem;
`;

export const StyledIcon = styled(Icon)`
  width: 1.4rem;
  fill: ${props => props.color};
  transition: 0.3s;
  flex: 1;
  margin-top: 0.2rem;
`;

export const IconContainer = styled.button`
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: ${props => props.color};
  outline-color: transparent;
`;

export const StyledParagraph = styled(Paragraph)`
  padding: 1rem 0;
`;

export const StyledCheckbox = styled(DefaultCheckbox)`
  font-size: ${F.SM};
  margin-bottom: 1rem;
`;

export const StyledTextArea = styled.textarea`
  background-color: ${C.GreyLight};
  padding: 1rem;
  color: ${C.GreySecondaryText};
  font-weight: ${F.SemiBold};
  font-size: ${F.MD};
  border: none;
  border-radius: 1rem;

  ::placeholder {
    color: ${C.GreySecondaryText};
    font-weight: ${F.SemiBold};
    font-size: ${F.MD};
    opacity: 0.5;
  }
`;

export const StyledRoundedButton = styled(RoundedButton)`
  font-weight: ${F.Bold};
  box-shadow: 0 6px 12px ${C.boxShadowColorLight};
`;
