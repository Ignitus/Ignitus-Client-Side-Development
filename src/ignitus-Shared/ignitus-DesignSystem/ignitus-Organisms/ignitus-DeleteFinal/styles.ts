import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import { Paragraph } from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';
import { Icon } from '../../../ignitus-Utilities/Components/icon';
import { RoundedButton } from '../../ignitus-Atoms/buttons';

export const DeleteContainer = styled.div`
  background-color: ${C.White};
  background-size: contain;
  border-radius: 1rem;
  display: inline-flex;
  flex-direction: column;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  width: 45rem;
  padding: 2rem;
`;

export const TopSection = styled(flexibleRowDiv)`
  justify-content: space-between;
`;

export const MiddleSection = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
`;

export const BottomSection = styled(flexibleRowDiv)`
  margin-top: 1rem;
  justify-content: flex-end;
`;

export const StyledIcon = styled(Icon)`
  width: 1.4rem;
  fill: ${props => props.color};
  transition: 0.3s;
  flex: 1;
  margin-top: 0.2rem;
`;

export const IconContainer = styled.button`
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: ${props => props.color};
  outline-color: transparent;
`;

export const StyledParagraph = styled(Paragraph)`
  padding: 0.5rem 0;
`;

export const StyledText = styled(Paragraph)`
  font-weight: ${F.Bold};
  padding: 0.5rem 0;
`;

export const StyledRoundedButton = styled(RoundedButton)`
  box-shadow: 0 6px 12px ${C.boxShadowColorLight};
`;
