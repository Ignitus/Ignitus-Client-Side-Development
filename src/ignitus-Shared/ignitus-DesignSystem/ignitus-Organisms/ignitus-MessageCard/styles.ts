import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import * as T from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';

export const MessageContainer = styled(flexibleRowDiv)`
  border-radius: 1rem;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  background-color: ${C.White};
  box-sizing: border-box;
  padding: 1rem;
  width: 100%;
  margin-bottom: 1rem;
`;

export const ImageContainer = styled.div`
  position: relative;
  padding: 0.5rem;

  img {
    border-radius: 50%;
    width: 4rem;
    max-width: 100%;
    max-height: 100%;
  }

  svg {
    position: absolute;
    bottom: 0;
    right: 0;
    stroke: ${C.White};
    fill: #219653;
    height: 2rem;
    width: 2rem;
  }
`;

export const MessageSubTitle = styled(T.Paragraph)`
  margin-left: 1rem;
`;

export const Sender = styled(T.Paragraph)`
  font-weight: ${F.Bold};
  font-size: ${F.MD};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const SenderPosition = styled(T.Paragraph)`
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const MessageDateTitle = styled.div`
  text-align: right;
  margin-left: auto;
`;

export const Details = styled(T.Paragraph)`
  font-family: Open Sans;
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.GreyOne};
  margin-bottom: 0.5rem;
`;
