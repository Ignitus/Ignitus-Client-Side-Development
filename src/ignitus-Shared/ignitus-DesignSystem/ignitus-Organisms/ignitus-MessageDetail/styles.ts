import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import * as T from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv, flexibleColDiv, Paragraph } from '../../shared';
import { DefaultButtonWithIcon } from '../../..';

export const MessageContainer = styled(flexibleColDiv)`
  border-radius: 1rem;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  background-color: ${C.White};
  box-sizing: border-box;
  width: 100%;
  margin-bottom: 1rem;
`;

export const ImageContainer = styled.div`
  position: relative;
  padding: 0.5rem;

  img {
    border-radius: 50%;
    width: 4rem;
    max-width: 100%;
    max-height: 100%;
  }

  svg {
    position: absolute;
    bottom: 0;
    right: 0;
    stroke: ${C.White};
    fill: #219653;
    height: 2rem;
    width: 2rem;
  }
`;

export const MessageSubTitle = styled(T.Paragraph)`
  margin-left: 1rem;
`;

export const Sender = styled(T.Paragraph)`
  font-weight: ${F.Bold};
  font-size: ${F.MD};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const SenderPosition = styled(T.Paragraph)`
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const MessageDateTitle = styled.div`
  text-align: right;
  margin-left: auto;
`;

export const Details = styled(T.Paragraph)`
  font-family: Open Sans;
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.GreyOne};
  margin-bottom: 0.5rem;
`;

export const Message = styled(Paragraph)``;

export const TopSection = styled(flexibleRowDiv)`
  box-sizing: border-box;
  padding: 1rem 1rem 0 1rem;
  width: 100%;
`;

export const MiddleSection = styled(flexibleRowDiv)`
  box-sizing: border-box;
  padding: 1rem;
  width: 100%;
`;

export const BottomSection = styled(flexibleRowDiv)`
  box-sizing: border-box;
  padding-right: 1rem;
  width: 100%;
  margin-bottom: 1rem;
  justify-content: flex-end;
`;

export const StyledDefaultButtonWithIcon = styled(DefaultButtonWithIcon)`
  flex-direction: row-reverse;
  box-shadow: 0 6px 12px ${C.boxShadowColorLight};
  border-radius: 1rem;
  font-weight: ${F.Bold};
  svg {
    margin-right: 0.2rem;
  }
`;

export const StyledLine = styled.hr`
  width: -webkit-fill-available;
`;
