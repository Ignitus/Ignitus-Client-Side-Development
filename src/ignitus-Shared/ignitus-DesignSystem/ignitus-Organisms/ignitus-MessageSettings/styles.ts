import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import * as F from '../../ignitus-Atoms/fonts';
import { RoundedButton } from '../../ignitus-Atoms/buttons';

export const Container = styled.div`
  background: ${C.White};
  box-shadow: 0px 3px 6px ${C.boxShadowColor};
  border-radius: 1rem;
  padding: 1.5rem;
  width: 40rem;
`;

export const StyledCheckboxContainer = styled.div`
  padding: 0.7rem 1rem;
`;

export const StyledButton = styled(RoundedButton)`
  background: ${C.SecondaryColor};
  font-size: ${F.MD};
  font-weight: ${F.Bold};
  color: ${C.White};
`;

export const StyledButtonContainer = styled.div`
  display: flex;
  flex-direction: row-reverse;
  padding-top: 2rem;
`;
