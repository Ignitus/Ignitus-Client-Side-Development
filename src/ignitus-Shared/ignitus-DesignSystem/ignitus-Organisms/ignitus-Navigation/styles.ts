import styled from '@emotion/styled';
import { Link } from 'react-router-dom';
import { keyframes, css } from '@emotion/core';
import isValidProps from '@emotion/is-prop-valid';
import {
  White,
  SecondaryColor,
  IgnitusBlue,
  boxShadowColor,
  Black,
} from '../../ignitus-Atoms/colors';
import { XS, SemiBold, Normal } from '../../ignitus-Atoms/fonts';
import { NavigationProps, ToggleProps, StyleProps } from './types';
import { Icon } from '../../../ignitus-Utilities/Components/icon';

export const Navigation = styled.nav<NavigationProps>`
  align-items: center;
  display: flex;
  position: fixed;
  width: 100%;
  top: 0;
  left: 0;
  z-index: 2;
  height: 3rem;
  justify-content: space-between;
  transition: background-color 300ms ease-in;
  background: ${props => (props.transparentNavigation ? 'transparent' : White)};
  box-shadow: ${props =>
    props.transparentNavigation ? 'none' : `0 2px 4px 0 ${boxShadowColor}`};
  padding: 0.5rem;
  @media only screen and (max-width: 1042px) {
    flex-direction: column;
    background: ${White};
  }
`;

export const NavigationBarBrand = styled(Link)`
  /* padding: 1rem; */
  align-self: stretch;
`;
export const Logo = styled(Icon)<StyleProps>`
  width: 2.5rem;
  height: 2.5rem;
  margin: ${props => (props.marginHorizontal ? '0rem 1.5rem 0rem 2.5rem' : '')};
  path {
    fill: ${IgnitusBlue};
  }
`;

const fadeIn = keyframes`
    from { opacity: 0; }
      to { opacity: 1; }
`;

const fadeOut = keyframes`
    from { opacity: 1; }
      to { opacity: 0; }
`;

export const NavigationLinks = styled.ul<ToggleProps>`
  display: flex;
  justify-content: center;
  align-items: center;
  margin-right: 1rem;

  @media only screen and (max-width: 1042px) {
    flex-direction: column;
    align-items: center;
    justify-content: center;
    width: 100%;
    display: ${props => (props.isExpanded ? 'none' : 'flex')};
    animation: ${props =>
      props.isExpanded
        ? css`
            ${fadeOut} 2s ease
          `
        : css`
            ${fadeIn} 2s ease
          `};
  }
`;

export const NavigationLinkItem = styled.li<NavigationProps>`
  padding: 0rem 0.5rem;
  align-self: stretch;
  display: flex;
  a {
    font-size: ${XS};
    font-weight: ${SemiBold};
    color: ${props => (props.transparentNavigation ? White : IgnitusBlue)};
    &:hover {
      color: ${SecondaryColor};
    }
    .home {
      color: ${Black};
      margin: 0 0.5rem;
      font-weight: ${Normal};
    }
  }

  @media only screen and (max-width: 1042px) {
    margin-left: 0rem;
    margin-bottom: 1rem;

    a {
      font-size: ${XS};
      font-weight: ${SemiBold};
      color: ${IgnitusBlue};
    }
  }
`;

export const NavigationLink = styled(Link)``;

export const Burger = styled(Icon, { shouldForwardProp: isValidProps })<
  ToggleProps
>`
  position: absolute;
  top: 1.2rem;
  right: 2rem;
  display: none;
  cursor: pointer;
  width: 2rem;
  transform: ${props => (props.isExpanded ? 'rotate(90deg)' : 'rotate(0deg)')};
  transition: transform 200ms ease-in-out;

  @media only screen and (max-width: 1042px) {
    display: block;
    top: 1.2rem;
  }
`;

export const StyledIcon = styled(Icon)<StyleProps>`
  width: 1.2rem;
  height: 1.2rem;
  fill: ${IgnitusBlue};
`;
