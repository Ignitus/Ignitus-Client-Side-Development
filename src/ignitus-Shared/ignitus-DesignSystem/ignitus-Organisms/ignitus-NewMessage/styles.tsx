import styled from '@emotion/styled';
import { StyledTagProps } from './types';
import { Paragraph, DefaultInput } from '../../..';
import { Bold } from '../../ignitus-Atoms/fonts';
import * as C from '../../ignitus-Atoms/colors';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';

export const Container = styled.div<StyledTagProps>`
  background: ${props => props.bgColor};
  color: ${props => props.color};
  border-radius: 0.5rem;
  margin: 0.5rem;
  display: flex;
  width: 50rem;
  height: 38rem !important;
  flex-direction: column;
  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
`;

export const IconContainer = styled.button`
  background-color: ${C.IgnitusBlue};
  border: none;
  cursor: pointer;
`;

export const Text = styled(Paragraph)`
  font-weight: ${Bold};
  color: ${props => props.color};
`;

export const MessageHeader = styled(flexibleRowDiv)`
  width: 90%;
  align-items: center;
  justify-items: center;
  margin: 10px 0px 10px 0px;
`;
export const HeaderLeft = styled.div`
  flex: 1;
  justify-content: flex-start;
  font-size: ${F.MD};
  padding-left: 12px;
  margin-left: 10px;
  color: #e0e0e0;
  line-height: ${F.LG};
`;

export const Input = styled(DefaultInput)`
  background-color: #f7f7f7;
  margin-left: 2rem;
  color: ${C.GreySecondaryText};
  font-weight: ${F.SemiBold};
  font-size: ${F.XL};
`;

export const BodyOptions = styled(flexibleRowDiv)<StyledTagProps>`
  background: ${C.IgnitusBlue};
  width: 104%;
  height: 44px;
  align-items: center;
  margin: 0 -1rem;
`;

export const BodyTab1 = styled.div`
  display: flex;
  height: 100%;
  flex: 1;
  flex-direction: row;
  align-items: center;
  justify-content: space-evenly;
`;
export const BodyTab2 = styled.div`
  display: flex;
  flex: 1;
  height: 100%;
  flex-direction: row;
  align-items: center;
  justify-content: space-evenly;
  border: 0.45px solid #e0e0e0;
`;
export const BodyTab3 = styled.div`
  display: flex;
  flex: 3;
  height: 100%;
  flex-direction: row;
  align-items: center;
  justify-content: space-evenly;
`;

export const BtnContainer = styled.button`
  display: flex;
  cursor: pointer;
  align-items: center;
  margin-left: 70px;
  background-color: #ffffff;
  border: 2px solid #ffffff;
  border-radius: 29px;
`;

export const BtnIconContainer = styled.div`
  margin-right: 5px;
`;
export const ButtonText = styled.div`
  font-family: ${F.PrimaryFamily};
  font-style: normal;
  font-weight: ${F.Bold};
  font-size: ${F.XS};
  line-height: ${F.XS};
  color: ${C.IgnitusBlue};
`;

export const MessageBody = styled.div`
  width: 100%;
  height: 80%;
`;

export const MessageInput = styled.textarea`
  height: 85%;
  width: 85%;
  margin: 20px 20px 0 20px;
  background-color: #f7f7f7;
  padding: 20px 0 0 20px;
  color: ${C.GreySecondaryText};
  font-weight: ${F.SemiBold};
  font-size: ${F.XL};
  border: none;
  resize: none;

  ::placeholder {
    color: ${C.GreySecondaryText};
    font-weight: ${F.SemiBold};
    font-size: ${F.SM};
    opacity: 0.3;
  }
`;

export const ActionButtons = styled.div`
  display: flex;
  margin-bottom: 10px;
  margin-right: 20px;
  justify-content: flex-end;
`;

export const ActionBtnContainer = styled.button`
  display: flex;
  cursor: pointer;
  align-items: center;
  background-color: #ffffff;
  border: 2px solid #ffffff;
  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
  border-radius: 50px;
  width: 130px;
  padding-left: 25px;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
`;

export const ActionBtnContainer2 = styled.button`
  display: flex;
  cursor: pointer;
  align-items: center;
  background-color: ${C.IgnitusBlue};
  border: none;
  padding: 10px;
  padding-left: 25px;
  width: 122px;
  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
  border-radius: 50px;
  margin-left: 18px;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
`;

export const ButtonText1 = styled.div`
  font-family: ${F.PrimaryFamily};
  font-style: normal;
  font-weight: ${F.Bold};
  font-size: ${F.MD};
  line-height: ${F.LG};
  color: ${C.IgnitusBlue};
`;

export const ButtonText2 = styled.div`
  font-family: ${F.PrimaryFamily};
  font-style: normal;
  font-weight: ${F.Bold};
  font-size: ${F.MD};
  line-height: ${F.LG};
  color: #ffffff;
`;
