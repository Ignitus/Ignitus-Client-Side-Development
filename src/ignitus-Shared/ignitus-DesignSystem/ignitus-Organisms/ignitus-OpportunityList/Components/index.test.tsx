import React from 'react';
import { mount, render } from 'enzyme';
import emotionSerializer, { matchers } from 'jest-emotion';
import { BtnWithIcon, Opportunity } from './index';
import * as S from '../styles';

expect.addSnapshotSerializer(emotionSerializer);
expect.extend(matchers);

it('should render', () => {
  const tree = render(<Opportunity />);
  expect(tree).toMatchSnapshot();
});

describe('<Opportunity />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<Opportunity />);
  });

  it('should render BtnWithIcon', () => {
    expect(wrapper.find(BtnWithIcon).length).toBe(1);
    expect(wrapper.find(S.BtnContainerIcon).length).toBe(1);
    expect(wrapper.find(S.BtnContainerText).length).toBe(1);
  });

  it('should render Opportunity', () => {
    wrapper = mount(<Opportunity />);

    expect(wrapper.find(Opportunity).length).toEqual(1);
    expect(wrapper.find('img').length).toEqual(1);
    expect(wrapper.find(S.OpportunityTitle).length).toEqual(1);
    expect(wrapper.find(S.OpportunityData).length).toEqual(1);
    expect(wrapper.find(S.OpportunitySubTitle).length).toEqual(1);
    expect(wrapper.find(S.OpportunityDescription).length).toEqual(1);
  });
});
