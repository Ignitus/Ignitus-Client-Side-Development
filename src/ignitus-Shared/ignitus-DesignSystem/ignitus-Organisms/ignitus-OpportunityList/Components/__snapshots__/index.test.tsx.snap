// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
.emotion-12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 4px;
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.05);
  background-color: #ffffff;
  padding: 1rem;
  width: 100%;
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex: 6;
  -ms-flex: 6;
  flex: 6;
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.emotion-0 img {
  max-width: 100%;
  max-height: 100%;
}

.emotion-10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 5;
  -ms-flex: 5;
  flex: 5;
  padding-left: 1rem;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-1 {
  font-weight: 500;
  font-size: 22px;
  color: #000066;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.emotion-2 {
  font-size: 16px;
  font-weight: 400;
  color: #2D2D2D;
  margin: 0;
  color: #2D2D2D;
  font-weight: 400;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  text-align: end;
}

.emotion-4 {
  font-size: 16px;
  font-weight: 400;
  color: #2D2D2D;
  margin: 0;
  font-weight: 400;
  color: #9C9C9C;
}

.emotion-9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-5 {
  font-size: 16px;
  font-weight: 400;
  color: #2D2D2D;
  margin: 0;
  font-weight: 400;
  color: #2D2D2D;
  margin-top: 1rem;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 5px 10px;
  border: 2px solid #000066;
  border-radius: 5px;
  cursor: pointer;
}

.emotion-6 {
  height: 1.5rem;
  fill: #000066;
}

.emotion-7 {
  font-size: 16px;
  font-weight: 300;
  font-size: 18px;
  font-weight: 400;
  margin-bottom: 0;
  margin-left: 10px;
  color: #000066;
}

<div
  class="emotion-12"
>
  <div
    class="emotion-11"
  >
    <div
      class="emotion-0"
    >
      <img
        alt="universityLogo"
        height="150"
      />
    </div>
    <div
      class="emotion-10"
    >
      <div
        class="emotion-3"
      >
        <h3
          class="emotion-1"
        />
        <p
          class="emotion-2"
        />
      </div>
      <p
        class="emotion-4"
      >
         |  
        <span
          aria-label="flag"
          role="img"
        >
          ðŸ‡©ðŸ‡ª
        </span>
      </p>
      <div
        class="emotion-9"
      >
        <p
          class="emotion-5"
        >
           
        </p>
        <div
          class="emotion-8"
        >
          <svg
            aria-label="Apply Icon"
            class="emotion-6"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Apply Icon
            </title>
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <path
              d="M18 19H6c-.55 0-1-.45-1-1V6c0-.55.45-1 1-1h5c.55 0 1-.45 1-1s-.45-1-1-1H5c-1.11 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-6c0-.55-.45-1-1-1s-1 .45-1 1v5c0 .55-.45 1-1 1zM14 4c0 .55.45 1 1 1h2.59l-9.13 9.13c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0L19 6.41V9c0 .55.45 1 1 1s1-.45 1-1V3h-6c-.55 0-1 .45-1 1z"
            />
          </svg>
          <p
            class="emotion-7"
          >
            Apply
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
